### ISSUE TITLE
Status of this project

### ISSUE THREAD

**Issue by spapas** [elton2048/wagtail-orderable#30]
Hello, I see that the latest commit is like 18 months ago. Is this project still supported? Can I run it with latest wagtail versions (2.14)? 

TIA

**Comment by th3hamm0r** [comment]
I'm using this [PR](https://github.com/elton2048/wagtail-orderable/pull/27) with the latest wagtail 2.14.1, and I looks good to me. The PR fixes some bugs and also includes helpful improvements.
Unfortunately the maintainer seems to be inactive atm, so if there are any bugs, you will probably have to fix it by yourself.
But I haven't found any better alternative for wagtail yet...

**Comment by spapas** [comment]
Thank you very much @th3hamm0r I'll also try this PR since it works with latest wagtail!

**Comment by elton2048** [comment]
Sorry for those who still using this package. I am not actively using wagtail atm so the package is not developed, and I aware of that situation now...

For short term I think I will read PR and try to have a more automatic deployment process, but not really try the feature. If anyone who interests to maintain the project I am happy to know.

**Comment by th3hamm0r** [comment]
Hi @elton2048, thanks for your response!
We're currently using #27 (which also fixes the issue of #31), and by only changing the wagtail-requirement in setup.py it also works without issues with the new wagtail 3.0. This has been solved (with other minor improvements) by the latest PR #32. 
So if we could try to create a new version (eg. 1.1) with those changes, that would be awesome. I would be happy to help with this.

**Comment by elton2048** [comment]
Hi there! I have merged some minor PRs as well as providing publish Github Actions for easier pipelining, so a new release is there. However #27 is not merged as there are some conflicts with some comments on the code so it is not yet addressed. It is good to have a new PR or update the old one to support that scenario. I will have a new release ASAP once that PR is ready
